[2022-08-19 23:10:55,868] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:32:00+00:00 [queued]>
[2022-08-19 23:10:55,887] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:32:00+00:00 [queued]>
[2022-08-19 23:10:55,888] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:55,889] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:10:55,891] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:55,910] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 06:32:00+00:00
[2022-08-19 23:10:55,932] {standard_task_runner.py:52} INFO - Started process 4281 to run task
[2022-08-19 23:10:55,951] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T06:32:00+00:00', '--job-id', '814', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmptfozq59h', '--error-file', '/tmp/tmpdowic_xu']
[2022-08-19 23:10:55,956] {standard_task_runner.py:80} INFO - Job 814: Subtask get_telegram_version
[2022-08-19 23:10:56,119] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:32:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:10:56,305] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T06:32:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T06:32:00+00:00
[2022-08-19 23:10:56,307] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:10:56,407] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T063200, start_date=20220819T231055, end_date=20220819T231056
[2022-08-19 23:10:56,483] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:10:56,521] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
